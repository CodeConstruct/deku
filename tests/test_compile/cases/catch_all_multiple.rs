use deku::prelude::*;

#[derive(DekuRead)]
#[deku(type = "u8")]
enum Test1 {
    #[deku(catch_all)]
    A = 1,
    #[deku(catch_all)]
    B = 2,
}

fn main() {}
