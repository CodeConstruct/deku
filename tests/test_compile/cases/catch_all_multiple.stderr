error: DekuRead: `catch_all` must be specified only one time
 --> tests/test_compile/cases/catch_all_multiple.rs:9:5
  |
9 |     B = 2,
  |     ^
